src/services/__tests__/redisSessionStorage.test.ts

```diff
@@ -386,31 +371,31 @@ describe("RedisSessionStorage#set", () => {
       expect(mockSet.mock.calls[0][0]).toBe(
         `SESSION-${aValidUser.session_token}`
       );
-      expect(mockSet.mock.calls[0][1]).toEqual(JSON.stringify(aValidUser));
```

Comment:
in the new redis sdk the value is passed as the second argument in a `setEx` method. same edit was applied for the few lines below